ironic_bug_dashboard/configs/ironic.json

```diff
@@ -0,0 +1,84 @@
+{
+    "projects": [
+        "bifrost",
+        "ironic",
+        "ironic-inspector",
+        "ironic-lib",
+        "ironic-prometheus-exporter",
+        "ironic-python-agent",
+        "ironic-python-agent-builder",
+        "ironic-ui",
+        "metalsmith",
+        "networking-baremetal",
+        "networking-generic-switch",
+        "nova",
+        "python-ironic-inspector-client",
+        "python-ironicclient",
+        "sushy",
+        "sushy-tools",
+        "virtualbmc",
+        "virtualpdu"
+    ],
+    "tagged_projects": [
+        {
+            "project_name": "nova",
+            "tags": "ironic"
+        }
+    ],
+    "priority_required_statuses": ["New", "In Progress", "Triaged", "Confirmed"],
+    "status_priorities": {
+        "In Progress": -10,
+        "Triaged": -5,
+        "Confirmed": -5
+    },
+    "useful_links": [
```

Comment:
I don't see how any of these are useful for a bug dashboard.

Reply:
Status priorities or useful links?

Reply:
I mean the links.